src/lib/intent-validator/tag-update.ts

```diff
@@ -1,6 +1,10 @@
 import SchemaValidator from './schema-validator'
 import * as Joi from 'joi'
 
+// TODO: We check this here and as part of the offline-api
+// validation. Which is the right place?
+const MAX_NAME_LENGTH = 256
```

Comment:
Seems good to catch this here as early as possible.

Reply:
I agree, but just wonder if we then need an offline-api schema validation at all for tags? I see that it makes sense for content types, because MAX_FIELDS have to be checked, but in the case of tags there really only is the name property. I think this is still my main uncertainty, how the separation of concerns between these two rounds of validations really work.